scripts/label_for_url.py

```diff
@@ -298,6 +298,26 @@ def determine_label(input_url: str) -> str:
         )
         page_id = f"/{m.group('pageId')}" if m.group("pageId") else ""
         return f"{space}{title}{anchor}" or f"{m.group('host')}{page_id}"
+    elif m := re.search(
+        r"^https://(?P<org>.*atlassian.*)"
+        r"/wiki/spaces/(?P<space>[^/]+)"
+        r"(?:/pages/(?P<pageId>.*/(?P<title>[^?#]+))?)?"
+        r"\??(?P<args>.*)",
+        input_url,
+    ):
+        space = m.group("space") if m.group("space") else ""
+        title = f"/{prettify(m.group('title'))}" if m.group("title") else ""
+        args = m.group("args").split("&")  # ["focusedCommentId=43245234", "test=abc"]
+
+        args = args[0].split("=") if args[0].startswith("focusedCommentId") else args
+        args = " > ".join(args) if args[0] == "focusedCommentId" else ""
+        args = (
+            args.replace("focusedCommentId >", " > Comment", 1)
+            if args.startswith("focusedCommentId >")
+            else args
```

Comment:
❔ I have trouble visualizing what `else args` results in. Can you please show me an example where this doesn't put Weird Shit™ into the label text? 🤔

I feel like the "default" should be `args == ""` unless there's some arg we specifically want to rescue (such as commend IDs).